components/menu/index.css

```diff
@@ -99,7 +99,7 @@ governing permissions and limitations under the License.
     }
   }
 
-  .spectrum-Icon {
+  .spectrum-Menu-itemIcon {
     /* Don't get smaller, you're an icon! */
     flex-shrink: 0;
     align-self: flex-start;
```

Comment:
There's another instance below that needs this change. I would say we should also preserve support for the existing markup so we don't have to make a breaking change.

Reply:
Ahh! Good catch. Fixed!